packages/textinput/src/css/index.js

```diff
@@ -44,7 +44,7 @@ export default {
     padding: `6px ${layout.spacingXSmall}`
   },
   '.psds-text-input__field--w-after': {
-    paddingRight: 0
+    paddingRight: layout.spacingXSmall
```

Comment:
Kinda odd.  I wonder why we would have set this explicitly as 0.  Did that value work at some previous point?